.github/workflows/eslint.yml

```diff
@@ -0,0 +1,27 @@
+name: ESLint
```

Comment:
This could be inlined as another step in your main `node.js` GitHub action

Reply:
I've decided to keep these as two separate files because this builds with node 16, and the node.js.yml builds with 14. Whenever I try using 14 in the eslint.yml file, the build fails.